extensions/pdh/tests/PerformanceDataMonitorTests.cpp

```diff
@@ -136,8 +136,9 @@ TEST_CASE("PerformanceDataMonitorCustomPDHCountersTest", "[performancedatamonito
 
```

Comment:
I have seen the assertion at line 127 fail seemingly randomly in some CI jobs (I think they were all Windows 2017 jobs).  Any idea why that happens?

Reply:
yeah, I already talked about that with @lordgamez , my guess is that the first flowfile can be invalid due to no wait time between the onSchedule and the onTrigger
Will definetly fix that, however I wouldnt delay this PR with that

Reply:
Agreed, it should be in a separate PR if it is not an obvious/tiny change.